conn.go

```diff
@@ -180,7 +206,26 @@ func (s *session) readPacket(ctx context.Context) ([]byte, error) {
 		return p, errInvalidSeqNo
 	}
 
-	crypt(p, s.c.Secret)
```

Comment:
I think it will work great when we read the packet. But when we write, we need to use the same secret that we chose among `RotatingSecrets`:
https://github.com/nwaples/tacplus/blob/master/conn.go#L200

Reply:
That's a great point. I'll push another revision to address that!

Reply:
The commit has been updated for encryption.